Riot/Modules/Room/TimelineDecorations/ReadReceipts/MXKReceiptSendersContainer.m

```diff
@@ -54,10 +55,12 @@ - (instancetype)initWithFrame:(CGRect)frame andMediaManager:(MXMediaManager*)med
     return self;
 }
 
-- (void)refreshReceiptSenders:(NSArray<MXRoomMember*>*)roomMembers withPlaceHolders:(NSArray<UIImage*>*)placeHolders andAlignment:(ReadReceiptsAlignment)alignment
+- (void)refreshReceiptSenders:(NSArray<MXRoomMember*>*)roomMembers nonRoomMembersReceiptSenders: (NSArray<MXUser*>*)nonRoomMembers withPlaceHolders:(NSArray<UIImage*>*)placeHolders andAlignment:(ReadReceiptsAlignment)alignment
 {
     // Store the room members and placeholders for showing in the details view controller
```

Comment:
We need to update this comment. Also lose the extra new line between the room and non-room members.